packages/linter/README.md

```diff
@@ -53,54 +53,29 @@ performed.
 
 ## Development
 
-**Important note!** Many of the scripts below rely on binaries that are
-installed in the `../../node_modules/.bin` directory, and _will fail_ if invoked
-in the default configuration. To fix this, either:
+### Commands/Scripts
 
-1. Add `../../node_modules/.bin` to your path. A tool like
-   [direnv](https://direnv.net/) may make this easier.
-2. Invoke via "wrapper" scripts in `../../package.json`. "test:node:linter" is
-   an example of such a script.
+#### `prepack`
 
-### Commands
+Intended to be used when building a package. (It's automatically invoked by `npm pack`.) Populates the `dist` directory with the appropriate `*.js` files. (Tests
```

Comment:
Is `npm pack` still a thing?

Reply:
I use it to ensure that the package itself works as expected (make sure there are no bugs in `package.json` making typings unavailable to dependent modules, say). There doesn't seem to be a way to get lerna to build the packages, but not publish them--?